src/mame/namco/namcos21.cpp

```diff
@@ -490,6 +494,7 @@ uint32_t namcos21_state::screen_update(screen_device &screen, bitmap_ind16 &bitm
 
 	m_namcos21_3d->copy_visible_poly_framebuffer(bitmap, cliprect, 0x7fc0, 0x7ffe);
 	m_namcos21_3d->copy_visible_poly_framebuffer(bitmap, cliprect, 0, 0x7fbf);
+
 	winrun_bitmap_draw(bitmap,cliprect);
```

Comment:
Same here.  These three lines are the final composite operation – it doesn’t help to break it up.

Reply:
Done.